app/lib/authenticated_system.rb

```diff
@@ -30,7 +30,6 @@ def clear_current_user
     authenticated_request.reset!
   end
 
-  # Feel freee to add skip_after_action for this method in controllers for stats or api transactions
```

Comment:
why remove this line?

Reply:
Yeah.. IDK, I thought this PR solved this comment but I guess it's still valid.